podpac/__init__.py

```diff
@@ -43,7 +43,7 @@ def makedirs(name, mode=511, exist_ok=False):
 from podpac.core.coordinates import Coordinates, crange, clinspace
 from podpac.core.node import Node, NodeException
 import podpac.core.authentication as authentication
-from podpac.core.utils import NodeTrait
```

Comment:
Me, too. This merge request ought to fix that. The NodeTrait is now a `tl.Instance` instead of a `tl.ForwardDeclaredInstance`, and it imports the `Node` class as needed.